support/cas-server-support-saml-idp-discovery/src/main/java/org/apereo/cas/web/SamlIdentityProviderDiscoveryFeedController.java

```diff
@@ -34,6 +36,7 @@
  * @since 6.1.0
  */
 @RestController("identityProviderDiscoveryFeedController")
+@ConditionalOnFeatureEnabled(feature = CasFeatureModule.FeatureCatalog.DelegatedAuthentication, module = "saml")
```

Comment:
This does not belong here.  Persumably, the configuration component that captures the bean definition should be carrying this, if not already